bzira.py

```diff
@@ -187,11 +188,14 @@ def parse_options():
     
 bz = bugzilla.Bugzilla(url=bzserver + "bugs/xmlrpc.cgi")
 
-print "[DEBUG] " + "Querying bugzilla: " + query
+
+if (options.verbose):
+    print "[DEBUG] " + "Querying bugzilla: " + query.replace("last_change_time","chfieldfrom")
```

Comment:
anyway, the query we actually send is what needs to be in the output - clickable link is secondary.